tools/xcodeproj-shims/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The new build system leaves a folder in the DerivedData directory which (for some reason) causes incremental build and test attempts during local development to fail with an error from Xcode at launch time. The error message says "Cannot attach to pid." This error seems to happen in the Xcode IDE, not when the project is tested from the xcodebuild command.